dotCMS/src/integration-test/java/com/dotcms/util/TestDataSourceProvider.java

```diff
@@ -63,18 +26,26 @@ public DataSource apply() {
      * Load properties file with DB connection details
      * @throws NamingException
      */
-    private void loadProperties()  {
-        ClassLoader loader = Thread.currentThread().getContextClassLoader();
-        prop = new Properties();
+    private Properties getProperties()  {
+        final ClassLoader loader = Thread.currentThread().getContextClassLoader();
+        final Properties properties = new Properties();
 
-        try (InputStream resourceStream = loader.getResourceAsStream("db-config.properties")) {
-            prop.load(resourceStream);
-        } catch (Exception e) {
-            Logger.info(this.getClass(), "Unable to get properties from file db-config.properties - let's try the environment");
+        String dbType = "postgres";
+
+        if (System.getProperty("databaseType")!=null){
```

Comment:
I guess you can just simply call:

`dbType = System.getProperty("databaseType", System.getenv("databaseType"))`

Reply:
Done